instances/technique/figure-groundVisualStimulation.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/figure-groundVisualStimulation",
```

Comment:
as above, without the "-":
https://openminds.ebrains.eu/instances/technique/figureGroundVisualStimulation